test/generation.test.ts

```diff
@@ -0,0 +1,65 @@
+import * as fc from '../src/index'
+import {it} from 'mocha'
+import {expect} from 'chai'
+import {FluentPick} from '../src/arbitraries'
+
+const prng = function (seed: number): () => number {
+  return () => {
+    return seed = seed * 16807 % 2147483647
+  }
+}
+
+function compareSamples(sample1: FluentPick<number>[], sample2: FluentPick<number>[]): boolean {
+  for (let i = 0; i < sample1.length; i++) {
+    if (sample1[i].value !== sample2[i].value || sample1[i].original !== sample2[i].original)
+      return false
+  }
+  return true
+}
```

Comment:
I'm sure `mocha` has functions to make deep comparisons of arrays of objects. Use it.